plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaIntegrationPushDown.java

```diff
@@ -101,70 +100,60 @@ public void testPartitionPushDown()
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp between timestamp '%s' and timestamp '%s'", topicNameCreateTime, recordMessage.getStartTime(), recordMessage.getEndTime()), 2, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp >= timestamp '%s' and _timestamp < timestamp '%s'", topicNameCreateTime, recordMessage.getStartTime(), recordMessage.getEndTime()), 2, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp = timestamp '%s'", topicNameCreateTime, recordMessage.getStartTime()), 0, session);
     }
 
     @Test
     public void testTimestampLogAppendModePushDown()
             throws Exception
     {
         RecordMessage recordMessage = createTimestampTestMessages(topicNameLogAppend);
-        DistributedQueryRunner queryRunner = getDistributedQueryRunner();
-        // ">= startTime" insure including index 2, "< endTime"  insure excluding index 4;
-        String sql = format(
-                "SELECT count(*) FROM default.%s WHERE _timestamp >= timestamp '%s' and _timestamp < timestamp '%s'",
-                topicNameLogAppend,
-                recordMessage.getStartTime(),
-                recordMessage.getEndTime());
+        Session session = getSession();
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp < timestamp '%s'", topicNameLogAppend, recordMessage.getEndTime()), 4, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp <= timestamp '%s'", topicNameLogAppend, recordMessage.getEndTime()), 4, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp > timestamp '%s'", topicNameLogAppend, recordMessage.getStartTime()), 997, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp >= timestamp '%s'", topicNameLogAppend, recordMessage.getStartTime()), 998, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp between timestamp '%s' and timestamp '%s'", topicNameLogAppend, recordMessage.getStartTime(), recordMessage.getEndTime()), 2, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp >= timestamp '%s' and _timestamp < timestamp '%s'", topicNameLogAppend, recordMessage.getStartTime(), recordMessage.getEndTime()), 2, session);
+        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _timestamp = timestamp '%s'", topicNameLogAppend, recordMessage.getStartTime()), 0, session);
+    }
 
+    private void assertProcessedInputPossitions(String sql, long expectedProcessedInputPositions, Session session)
```

Comment:
Let's add a preparatory commit to fix typo. `assertProcessedInputPositions`

Reply:
I changed to assertProcessedInputPositions

Reply:
> Let's add a preparatory commit to fix typo.

This commend isn't addressed.

Reply:
I added another commit to fix it.